plugins/balloonpanel/plugin.js

```diff
@@ -320,10 +320,13 @@
 		 *
 		 * @method attach
 		 * @param {CKEDITOR.dom.element} element The element to which the panel is attached.
-		 * @param {CKEDITOR.dom.element/Boolean} [focusElement] The element to be focused after the panel
+		 * @param {Object} options Ballonpanel focus and show options
+		 * @param {CKEDITOR.dom.element/Boolean} [options.focusElement] The element to be focused after the panel
 		 * is attached. By default the `panel` property of {@link #parts} will be focused. You might specify the element
 		 * to be focused by passing any {@link CKEDITOR.dom.element} instance.
 		 * You can also prevent changing focus at all by setting it to `false`.
+		 * Balloonpanel still support old API where focusElement was second param of attach method.
```

Comment:
This should be actually documented on `options` param itself, which will be tricky to put in a simple way, as it will be something like:

```
		 * @param {Object/CKEDITOR.dom.element/Boolean} options **Since 4.8.0** this parameter works as options object.
		 * If `CKEDITOR.dom.element/Boolean` instance is passed it's used as `options.focusElement`.
```

But it's still very little readable. Think how to simplify it, you can also ask Anna for help.